src/index.html

```diff
@@ -5,8 +5,247 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Catalog</title>
     <link rel="stylesheet" href="./styles/main.scss">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <a href="#" class="header__logo">
+        <img
+          class="header__logo-image"
+          src="images/logo.png"
+          alt="logo">
+      </a>
+      <nav>
+        <ul class="header__nav-list">
+          <li class="header__nav-item">
+            <a href="#" class="header__nav-link is-active">
+              Apple
+            </a>
+          </li>
+          <li class="header__nav-item">
+            <a href="#" class="header__nav-link">Samsung</a>
+          </li>
+          <li class="header__nav-item">
+            <a href="#" class="header__nav-link">Smartphones</a>
+          </li>
+          <li class="header__nav-item">
+            <a href="#"
+               data-qa="nav-hover"
+               class="header__nav-link">
+              Laptops & Computers
+            </a>
+          </li>
+          <li class="header__nav-item">
+            <a href="#" class="header__nav-link">Gadgets</a>
+          </li>
+          <li class="header__nav-item">
+            <a href="#" class="header__nav-link">Tablets</a>
+          </li>
+          <li class="header__nav-item">
+            <a href="#" class="header__nav-link">Photo</a>
+          </li>
+          <li class="header__nav-item">
+            <a href="#" class="header__nav-link">Video</a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+    <main class="cards">
+      <div data-qa="card" class="card">
+        <div class="card__picture"></div>
+        <div class="card__description">APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)</div>
+        <div class="card__product-code">Product code: 195434</div>
+        <div class="card__reviews">
+          <div class="card__reviews-stars">
+            <div class="card__reviews-star"></div>
+            <div class="card__reviews-star"></div>
+            <div class="card__reviews-star"></div>
+            <div class="card__reviews-star"></div>
+            <div class="card__reviews-star"></div>
+          </div>
+          <div class="card__reviews-text">Reviews: 5</div>
+        </div>
+        <div
+          class="card__price">
+          Price: <span class="card__price-amount">$2,199</span>
+        </div>
+        <a
+          data-qa="card-hover"
+          class="card__button card__button--active">
+          buy
+        </a>
```

Comment:
As above, format according to guideline line-wrapping

Reply:
To be honest, I'm not sure how it should be formatted now because it was previously accepted as the correct way to do it.